src/routes/LyntContents.svelte

```diff
@@ -232,7 +232,7 @@
 	</div>
 </div>
 {#if has_image}
-	<div class="max-h-[600px]">
```

Comment:
The height limit is probably there because people could make a really large image so it shouldn't make the page take years to scroll past the image.

Reply:
I don't know if there is a height limit for the content I think it's just the image but I probably could increase it

Reply:
I don't think it matters too much though the image is still gonna be the same aspect ratio it just be shrunken a little to fit